### DISCUSSION TITLE
聊聊区块链：IPFS 与 FileCoin · /ipfs-filecoin

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#524] [Announcements]
https://draveness.me/ipfs-filecoin

**Comment by daijiale** [comment]
归纳得很系统👍

**Comment by TimorYang** [comment]
利用这套系统可以解决高质量(蓝光)影片的存储，以前是想看这部影片，必须要下载到本地观看，那就意味着看过这部影片的用户们本地都存储着这部影片，这其实是对存储资源的浪费，利用IPFS，其实可以将影片分成若干个小块存储在各个节点中，当任一用户需要观看这部电影的时候，只需要发起请求，就可以从各个节点中获取到该影片，这样理论上，一部影片只需要存在一份，就可以供所有的节点用户观看。

**Comment by zjdznl** [comment]
感谢楼主的总结！看了一下，主要内容来自于[这篇paper](https://github.com/ipfs/ipfs/blob/master/papers/ipfs-cap2pfs/ipfs-p2p-file-system.pdf?raw=true)

**Comment by Redwinam** [comment]
楼主看好其他的分布式存储项目，例如storj v3吗？https://ipfsdrop.com/tech/xianduweikuaistorj-v3-baipishufabu/

**Comment by draveness** [comment]
> @Redwinam
> 楼主看好其他的分布式存储项目，例如storj v3吗？https://ipfsdrop.com/tech/xianduweikuaistorj-v3-baipishufabu/

不看好。。

**Comment by ace1573** [comment]
> @TimorYang
> 利用这套系统可以解决高质量(蓝光)影片的存储，以前是想看这部影片，必须要下载到本地观看，那就意味着看过这部影片的用户们本地都存储着这部影片，这其实是对存储资源的浪费，利用IPFS，其实可以将影片分成若干个小块存储在各个节点中，当任一用户需要观看这部电影的时候，只需要发起请求，就可以从各个节点中获取到该影片，这样理论上，一部影片只需要存在一份，就可以供所有的节点用户观看。

需要做冗余和扩散机制，因为分布式系统下节点是不稳定的。

**Comment by littleboss** [comment]
请问，IPFS的内容监管难，会不会阻碍它的发展。ps：作图是用啥软件？
